### PR TITLE
Add new color variables

### PR DESCRIPTION
**Pull request by maya**
This adds in our new color variables to the site :rainbow:

### CODE REVIEW

**Comment by maya** [comment]
I want to align the variables better, but some are really short and some are really long so you lose track of them bc they're so far apart.

File: assets/_scss/core/_variables.scss [hunk 1]
```diff
@@ -18,7 +18,52 @@ $font-serif:        'Georgia', 'Times New Roman', serif;
...
+$color-green-light:        lighten($color-green, 20%); // #649967
+$color-green-lighter:      lighten($color-green, 60%); // #94bfa2
+$color-green-lightest:     lighten($color-green, 80%); // #e7f4e4
+
+$color-cool-blue:          #205493;
+$color-cool-blue-light:    lighten($color-cool-blue, 20%); // #4773aa
+$color-cool-blue-lighter:  lighten($color-cool-blue, 60%); // #8ba6ca
+$color-cool-blue-lightest: lighten($color-cool-blue, 90%); // #dce4ef
+
 $color-grey-50:     #fafafa;
```
Comment (msecret) [inline assets/_scss/core/_variables.scss]: Are we going to get rid of the rest of these colors once we remove the references in the code?

If so, could you add a "deprecated" comment?

Comment (maya) [inline assets/_scss/core/_variables.scss]: Do you think there's any value for leaving in the grays?

Comment (maya) [inline assets/_scss/core/_variables.scss]: Removed after discussion

**Pull request merged**